examples/ota-requestor-app/esp32/main/main.cpp

```diff
@@ -73,6 +74,33 @@ static void InitServer(intptr_t context)
 
 } // namespace
 
+// Test mode operation
+void OnStartDelayTimerHandler(Layer * systemLayer, void * appState)
+{
+    OTARequestor * gRequestorCore = static_cast<OTARequestor *>(appState);
```

Comment:
nit: The `g` in gRequestorCore implies global (which is already defined above in line 57). This is a local variable so `requestorCore` would be more appropriate and to differentiate from the global that already exists.

Reply:
Done